src/wallet/wallet.cpp

```diff
@@ -1797,6 +1798,10 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
             }
         }
     }
+    if (!max_height) {
+        WalletLogPrintf("Scanning current mempool transactions.\n");
```

Comment:
I don't see this notice in the log either.

Reply:
Strange, that may be a hint that the build didn't have this change? But I will go through your steps as well as an additional check.

Reply:
Indeed it turns out I was running the wrong build (again). Let me try again. At least I know for sure the bug exists on master 506d9b25a3f8ef8a4ce63f989083803d8d4ae6df :-)